internal/limits.go

```diff
@@ -118,15 +137,35 @@ func (i *Internal) validateJob(job *model.Job) error {
 	// Verify that the user hasn't exceeded their limit for the number of concurrent jobs.
 	jobCount, err := i.countJobsForUser(user)
 	if err != nil {
-		return errors.Wrapf(err, "unable to determine the number of jobs the %s is currently running", user)
+		return http.StatusInternalServerError, errors.Wrapf(err, "unable to determine the number of jobs the %s is currently running", user)
 	}
 	jobLimit, err := i.getJobLimitForUser(user)
 	if err != nil {
-		return errors.Wrapf(err, "unable to determine the concurrent job limit for %s", user)
+		return http.StatusInternalServerError, errors.Wrapf(err, "unable to determine the concurrent job limit for %s", user)
+	}
+	defaultJobLimit, err := i.getDefaultJobLimit()
+	if err != nil {
+		return http.StatusInternalServerError, errors.Wrapf(err, "unable to determine the default concurrent job limit")
 	}
```

Comment:
I ended up using separate queries for the user's job limit and the default job limit. I could have done it in a single query, but this seemed cleaner to me. The only argument _for_ using a single query is that it might be marginally faster. I really wouldn't expect to see a significant enough performance gain from combining the queries to make a difference, though.